core/test/financial_templates/ExpiringMultiParty.js

```diff
@@ -1,29 +1,35 @@
+const { toWei } = web3.utils;
+
+// Tested Contract
 const ExpiringMultiParty = artifacts.require("ExpiringMultiParty");
-const Finder = artifacts.require("Finder");
+
 // Helper Contracts
+const Finder = artifacts.require("Finder");
 const ERC20MintableData = require("@openzeppelin/contracts/build/contracts/ERC20Mintable.json");
 const truffleContract = require("@truffle/contract");
 const ERC20Mintable = truffleContract(ERC20MintableData);
 ERC20Mintable.setProvider(web3.currentProvider);
 
 contract("ExpiringMultiParty", function(accounts) {
-  it("Empty", async function() {
-    finder = await Finder.new();
+  it("Can deploy", async function() {
+    const collateralToken = await ERC20Mintable.new({ from: accounts[0] });
+    const finder = await Finder.new();
```

Comment:
I know you didn't change this line, but could we make this `await Finder.deployed()` instead? Just wondering actually.

Reply:
So this touches on a larger question on how we want to orchestrate tests. If we use `.deployed()` it will pull from the truffle migration version, as done in other tests within the stack. However, this makes the tests dependent on the migrations which right now the rest of these new tests are not.

Ultimately we need to decide if we want to deploy these helper like contracts in migrations then import like this with `deployed()` or re-create the contract setup in every test. There are tradeoffs between speed and reproducibility here.

Reply:
In #869, I move all of these over to `Finder.deployed()` because that PR requires the Finder to link the contract to the Store, etc.